package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@tresjs/nuxt",
   "type": "module",
   "version": "3.0.8",
-  "packageManager": "pnpm@9.15.4",
```

Comment:
why removing pnpm as package manager?

can you add back the entry using latest 10.7.1 and run `pnpm install && pnpm approve-builds`? Will require to install it via corepack globally `corepack install -g pnpm@10.7.0` or `corepack install -g pnpm@10.7.0`

Reply:
No idea, it confused me even more knowing the repo uses pnpm. RFC unclear. Lemme modify.

Reply:
the `pnpm.*` entries added by pnpm when you run  `pnpm approve-builds` and selected the deps?